.github/workflows/docker-publish.yaml

```diff
@@ -25,6 +25,13 @@ jobs:
     - name: Set env
       # github tag will be like refs/tags/v0.1.0 :10 removes the first 10 characters
       run: echo RELEASE_VERSION=$(echo ${GITHUB_REF:10}) >> $GITHUB_ENV
+
+    - name: Set up QEMU
+      uses: docker/setup-qemu-action@v1
+      
+    - name: Set up Docker Buildx
+      id: buildx
+      uses: docker/setup-buildx-action@v1.3.0
```

Comment:
use latest `1.6.0` version here and in  other place also

Reply:
done